src/object/srv_ec.c

```diff
@@ -70,11 +76,20 @@ obj_ec_rw_req_split(struct obj_rw_in *orw, struct obj_ec_split_req **split_req)
 
 	req_size = roundup(sizeof(struct obj_ec_split_req), 8);
 	iods_size = roundup(sizeof(daos_iod_t) * iod_nr, 8);
-	D_ALLOC(buf, req_size + iods_size);
+	if (with_csums) {
+		csums_size = roundup(sizeof(struct dcs_iod_csums) * iod_nr, 8);
+		singv_ci_size = roundup(sizeof(struct dcs_csum_info) * iod_nr,
+					8);
```

Comment:
Just curious, what does this 8 means?

Reply:
just to make it better aligned, probably unnecessary for malloc.